### PR TITLE
feat: check that instructions were utilized in testbench

### PR DESCRIPTION
**Pull request by ddelgrosso1**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-storage/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: conformance-test/retryStrategy.ts [hunk 1]
```diff
@@ -19,12 +19,14 @@ import * as path from 'path';
 import * as uuid from 'uuid';
 import pLimit = require('p-limit');
 import * as assert from 'assert';
+import fetch from 'node-fetch';
```
Comment (shaffeeullah) [inline conformance-test/retryStrategy.ts:22]: does this introduce a dev dependency on node-fetch?

File: conformance-test/retryStrategy.ts [hunk 2]
```diff
@@ -84,6 +85,10 @@ const OPTIONS = {
   },
 };
 
+const EMULATOR_HOST =
+  process.env.STORAGE_EMULATOR_HOST || 'http://localhost:9000';
```
Comment (shaffeeullah) [inline conformance-test/retryStrategy.ts:89]: does this need a trailing `/`?

File: conformance-test/retryStrategy.ts [hunk 3]
```diff
@@ -84,6 +85,10 @@ const OPTIONS = {
   },
 };
 
+const EMULATOR_HOST =
```
Comment (shaffeeullah) [inline conformance-test/retryStrategy.ts:88]: should we call this `TESTBENCH_HOST`?

File: conformance-test/retryStrategy.ts (hunk 1 above)
Comment (ddelgrosso1) [reply conformance-test/retryStrategy.ts:22]: We already had `node-fetch` listed as a dev dependency in the `package.json` which is why I went with it as opposed to the `https` lib in node.

**Review by shaffeeullah — APPROVED** [review]

**Pull request merged**